azure-pipelines.yml

```diff
@@ -0,0 +1,84 @@
+trigger:
+- dev
+
+variables:
+    DOTNET_VERSION: '2.1.402'
+
+jobs:
+- job: Windows
+  pool:
+    vmImage: 'vs2017-win2016'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+      Python37:
+        python.version: '3.7'
+    maxParallel: 1
+
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(python.version)'
+      addToPath: true
+  - task: BatchScript@1
+    inputs:
+      filename: .ci\win_devops_tools.bat
+    displayName: 'Install Core Tools'
+  - task: BatchScript@1
+    inputs:
+      filename: .ci\win_devops_build.bat
+    displayName: 'Build'
+  - task: BatchScript@1
+    inputs:
+      filename: .ci\win_devops_tests.bat
+    env:
+      AzureWebJobsStorage: $(WindowsStorageConnectionString)
+      AzureWebJobsCosmosDBConnectionString: $(WindowsCosmosDBConnectionString)
+      AzureWebJobsEventHubConnectionString: $(WindowsEventHubConnectionString)
+      AzureWebJobsServiceBusConnectionString: $(WindowsServiceBusConnectionString)
+    displayName: 'Test'
+
+- job: Linux
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+      Python37:
+        python.version: '3.7'
+    maxParallel: 1
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(python.version)'
+      addToPath: true 
+  - task: ShellScript@2
+    inputs:
+      disableAutoCwd: true # Execute in current directory
+      scriptPath: .ci/linux_devops_tools.sh
+    displayName: 'Install Core Tools'
+  - task: ShellScript@2
+    inputs:
+      disableAutoCwd: true 
+      scriptPath: .ci/linux_devops_dotnet.sh
+    displayName: 'Install dotnet'
+  - task: ShellScript@2
+    inputs:
+      disableAutoCwd: true 
+      scriptPath: .ci/linux_devops_build.sh
+    displayName: 'Build'
+  - bash: |
+        chmod +x .ci/linux_devops_tests.sh
+        .ci/linux_devops_tests.sh
+    env:
+      LINUXSTORAGECONNECTIONSTRING: $(LinuxStorageConnectionString)
+      LINUXCOSMOSDBCONNECTIONSTRING: $(LinuxCosmosDBConnectionString)
+      LINUXEVENTHUBCONNECTIONSTRING: $(LinuxEventHubConnectionString)
+      LINUXSERVICEBUSCONNECTIONSTRING: $(LinuxServiceBusConnectionString)
+    displayName: 'Test'
```

Comment:
Nit: Test_Linux

Reply:
This should be fine. It's easy to see from the context in the yaml file as well as the UI which system is being tested.